.travis.yml

```diff
@@ -3,9 +3,21 @@ language: python
 addons:
   chrome: stable
 
-python:
-    - '3.6'
-    - '3.7'
+matrix:
+  include:
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8
```

Comment:
should py38 be part if the `envlist` in `tox.in` if we want to also test for py38

Reply:
Maybe, `TOXENV` has the highest priority when running `tox`, over `-e <param>`, meaning that travis will not run the full `envlist`, only one test per travis instance, but will run py38 even though it´s not specified in `envlist`. The reason I only set up one version in `envlist` is convenience when running locally. My reasoning is that py37 is the main development version, and running `tox` locally will run the main test suite, while some additional tests are performed by `travis`. That could be wrong though and I´ll be happy to have that discussion.